tasks/configure-docker.yml

```diff
@@ -0,0 +1,22 @@
+# file: postgresql/tasks/configure-docker.yml
+
+- name: PostgreSQL | Update configuration - pt. 1 (pg_hba.conf)
+  template:
+    src: pg_hba.conf.j2
+    dest: "{{postgresql_conf_directory}}/pg_hba.conf"
+    mode: 0640
```

Comment:
With a single file, I would have also to add `when` clauses to all the other tasks of the original `configure.yml` file. I think that it is cleaner this way, with the differentiation clearly marked in the `main.yml`.

It's also easier to maintain/integrate with the upstream repo.